static/scripts/administration.js

```diff
@@ -138,6 +141,7 @@ $(document).ready(() => {
 			title: $t('administration.school.headline.addPolicy'),
 			closeLabel: $t('global.button.cancel'),
 			submitLabel: $t('global.button.add'),
+			submitDataTestId: 'add-modal--policy',
```

Comment:
in general also here, please dont use mixture of hivens and double-hivens

Reply:
fixed